presentation/src/main/java/com/cointrend/presentation/ui/about/AboutScreen.kt

```diff
@@ -0,0 +1,236 @@
+                        onDoubleTap = { focusManager.clearFocus() }
+                    )
+                }
+        ) {
+            LazyColumn() {
+                // The Logp
+                item {
+                    CoinTrend()
+                }
+
+                // Setion title 16dp SOURCE CODE
+                item {
+                    SectionTitle(title = "Source Code", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Github", info = "https://github.com/CoinTrend/CoinTrend", image = R.drawable.ic_github, showDivider = false)
+                    }
+                }
+                item {
+                    Spacer(modifier = Modifier.size(16.dp))
+                }
+                // Setion title 16dp CONTACT
+                item {
+                    SectionTitle(title = "Contact", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Email", info = "cointrend.info@gmail.com", image = R.drawable.ic_gmail, showDivider = true)
+                        SectionInfoItemTest(name = "Github Issues", info = "https://github.com/CoinTrend/CoinTrend/issues", image = R.drawable.ic_github, showDivider = false)
+                    }
+                }
+                item {
+                    Spacer(modifier = Modifier.size(16.dp))
+                }
+                // Setion title 16dp SUPPORT
+                item {
+                    SectionTitle(title = "Support", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Donate Bitcoin", info = "https://github.com/CoinTrend/CoinTrend#bitcoin", image = R.drawable.ic_donate, showDivider = true)
```

Comment:
The correct URL is 'https://github.com/CoinTrend#support' and the title is just 'Donate'